src/users/users.controller.spec.ts

```diff
@@ -653,7 +652,6 @@ describe('UsersAdminController', () => {
       const statusDto = { status: UserStatus.ACTIVE };
       const mockReq = { user: createUserMock() };
 
-      // Mocking the service to throw an error when the user is not found
```

Comment:
Question: Why was there a removal of a comment that explained the code? @oyedeletemitope